src/index.tsx

```diff
@@ -176,6 +176,23 @@ if (
 ) {
   render();
   log.setDefaultLevel(log.levels.DEBUG);
+
+  if (process.env.NODE_ENV === `development`) {
+    settings.then((settingsResult) => {
+      if (settingsResult) {
+        const apiUrl = settingsResult.apiUrl;
+        axios
+          .post(`${apiUrl}/login`, {
+            username: 'frontend',
+            password: 'front',
+          })
+          .then((response) => {
+            window.localStorage.setItem(MicroFrontendToken, response.data);
```

Comment:
It is worth noting that this differs from DG plugins, which consists of:
```
const jwtHeader = { alg: 'HS256', typ: 'JWT' };
const payload = {
    sessionId: response.data.sessionID,
    username: 'Thomas409',
};
const jwt = jsrsasign.KJUR.jws.JWS.sign(
    'HS256',
    jwtHeader,
    payload,
    'shh'
);

window.localStorage.setItem(MicroFrontendToken, jwt);
```
Is the above still necessary here?

Reply:
No. Previously, we needed to do that as we logged into ICAT, which gave us a session ID - but we need to have a JWT in DG. So in dev we sign our own JWT and put the ICAT info in it, which is normally scigateway-auth's job. Now, since in dev we're logging into og-api which returns a JWT, we can just use that.